lstchain/reco/dl0_to_dl1.py

```diff
@@ -295,7 +294,39 @@ def r0_to_dl1(input_filename=get_dataset_path('gamma_test_large.simtel.gz'),
                         dl1_container.fill_mc(event)
 
                     dl1_container.log_intensity = np.log10(dl1_container.intensity)
-                    dl1_container.gps_time = event.trig.gps_time.value
+                    # dl1_container.gps_time = event.trig.gps_time.value
+
+                    # GPS time is not available for the time being in the above field.
+                    # In the mean time, it is taken from UCTS timestamp (in unix time)
+                    # or alternatively from the TIB pps and 10 MHz counters. The first
+                    # one started to work around mid Nov 2019, so previous runs take
+                    # timing information from the TIB NTP counters.
+                    # This will be deprecated and modified back to the commented line
+                    # whenever GPS time is dumped to gps_time field.
+
+                    if not is_simu:
+                        ucts_timestamp = event.lst.tel[telescope_id].evt.ucts_timestamp
+ 
+                        if ucts_timestamp != 0:
+                            ns = 1e-9  # Convert nanosecs to secs
```

Comment:
Using astropy units has been on the todo list for a long time.
Starting now would be great :)

Reply:
done